README.adoc

```diff
@@ -35,6 +36,33 @@ NOTE: If you want to use reveal.js 2, please switch your asciidoctor-reveal.js r
 
 TIP: If you are using https://pages.github.com/[GitHub Pages], plan ahead by keeping your source files on `master` branch and all output files on the `gh-pages` branch.
 
+[[node-package]]
+== Node package
+
+Using npm:
+
+ $ npm i -g npm
```

Comment:
I took inspiration from https://github.com/lodash/lodash#installation, the only hard requirement is to have npm available on your PATH. So basically if you install Node.js you are good to go.